CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 None
 
+<a name="5.3.1"></a>
+
+## [5.3.1] 2023-05-25
+
+### Changed
+
+- Resolve ApiClient class bug for requests
```

Comment:
This does not indicate what actually changed.

Reply:
Will update changelog statement with correct description of change in latest commit